pkgs/development/tools/alire/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, gnat, gprbuild }:
+
+stdenv.mkDerivation {
+  pname = "alire";
+  version = "1.3-dev";
+  src = fetchFromGitHub {
+    owner = "alire-project";
+    repo = "alire";
+    rev = "8f44a810";
```

Comment:
Why did you choose this commit?

Reply:
I chose this commit because the latest stable version has a broken version of gnatcoll-core that prevents it from building. Since Alire uses a slimmed-down version of gnatcoll-core with the fix already applied, it seemed pointless to manually patch out the relevant problems.